.editorconfig

```diff
@@ -1,10 +1,20 @@
 root = true
 
+
+[*]
+indent_style=space
+indent_size=2
```

Comment:
I think `rustfmt` uses 4 spaces.

Reply:
Right, we are keeping it 4 for *.rs files